code/game/turfs/simulated/floor/plating.dm

```diff
@@ -270,6 +270,11 @@
 	if (AM)
 		thing_to_check = list(AM)
 	for(var/thing in thing_to_check)
+		if(ismob(thing))
+			if(ishuman(thing))
+				var/mob/living/carbon/human/T = thing
+				if(FLYING in T.dna.species.specflags)
```

Comment:
Nooooooo. Bad. I specifically made wings not be lava proof because feathers. Also subtle Icarus flew too close to the sun joke.